doc/source/serve/doc_code/getting_started/fastapi_model.py

```diff
@@ -0,0 +1,64 @@
+# flake8: noqa
+
+# Stub functions for testing
+import ray
+from ray import serve
+
+ray.init()
+serve.start()
+
+
+def stub(*args, **kwargs):
+    pass
+
+
+ray.init = stub
+serve.start = stub
+
+# __fastapi_start__
+# File name: serve_with_fastapi.py
+import ray
+from ray import serve
+from fastapi import FastAPI
+from transformers import pipeline
+
+app = FastAPI()
+
+
+@serve.deployment
+@serve.ingress(app)
+class Translator:
+    def __init__(self):
+        # Load model
+        self.model = pipeline("translation_en_to_fr", model="t5-small")
+
+    @app.get("/translate")
+    def translate(self, text: str) -> str:
+        # Run inference
+        model_output = self.model(text)
+
+        # Post-process output to return only the translation text
+        translation = model_output[0]["translation_text"]
+
+        return translation
+
+
+ray.init(address="auto")
+serve.start(detached=True)
```

Comment:
we do not need `serve.start` anymore when using `serve.run (indeed it is deprecated). please update relevant code samples

Reply:
you can also drop the `ray.init` call

Reply:
I dropped `serve.start()`, but I kept `ray.init()` for now. I think it's useful to have users explicitly start a Ray cluster and connect to it in the tutorial to help them build a clearer mental model of how Ray Serve interacts with Ray. What do you think?